### PR TITLE
Add extension that implements SplitV.

### PR DESCRIPTION
**Pull request by pschuh**

### CODE REVIEW

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

File: stdlib/public/TensorFlow/ArrayOps.swift [hunk 1]
```diff
@@ -109,4 +109,50 @@ public extension Raw {
...
+  ///
+  /// - Parameters:
+  ///   - value: The tensor to split.
+  ///   - size_splits: list containing the sizes of each output tensor along the split
+  ///     dimension. Must sum to the dimension of value along split_dim.
+  ///     Can contain one -1 indicating that dimension is to be inferred.
+  ///   - split_dim: 0-D.  The dimension along which to split.  Must be in the range
+  ///     `[-rank(value), rank(value))`.
+  ///
+  /// - Output output: Tensors whose shape matches that of `value`
```
Comment (rxwei) [inline stdlib/public/TensorFlow/ArrayOps.swift]:
```suggestion
  /// - Returns: Tensors whose shape matches that of `value`
```

File: stdlib/public/TensorFlow/ArrayOps.swift [hunk 2]
```diff
@@ -109,4 +109,50 @@ public extension Raw {
     }
     return out
   }
+
+  /// Splits a tensor into `num_split` tensors along one dimension.
+  ///
+  /// - Parameters:
+  ///   - value: The tensor to split.
+  ///   - size_splits: list containing the sizes of each output tensor along the split
```
Comment (rxwei) [inline stdlib/public/TensorFlow/ArrayOps.swift]:
```suggestion
  ///   - sizeSplits: list containing the sizes of each output tensor along the split
```

File: stdlib/public/TensorFlow/ArrayOps.swift [hunk 3]
```diff
@@ -109,4 +109,50 @@ public extension Raw {
...
+  ///     dimension. Must sum to the dimension of value along split_dim.
+  ///     Can contain one -1 indicating that dimension is to be inferred.
+  ///   - split_dim: 0-D.  The dimension along which to split.  Must be in the range
```
Comment (rxwei) [inline stdlib/public/TensorFlow/ArrayOps.swift]:
```suggestion
  ///   - splitDim: 0-D.  The dimension along which to split.  Must be in the range
```

File: stdlib/public/TensorFlow/ArrayOps.swift [hunk 4]
```diff
@@ -109,4 +109,50 @@ public extension Raw {
     }
     return out
   }
+
+  /// Splits a tensor into `num_split` tensors along one dimension.
```
Comment (rxwei) [inline stdlib/public/TensorFlow/ArrayOps.swift]:
```suggestion
  /// Splits a tensor into `numSplit` tensors along one dimension.
```

**Review by rxwei — APPROVED** [review]

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by pschuh** [comment]
@swift-ci please test tensorflow

**Comment by rxwei** [comment]
Tested locally. Merging for v0.3.

**Pull request merged**